tools/run-dev.py

```diff
@@ -147,7 +147,8 @@
 if not os.path.exists("node_modules/.bin/handlebars") and os.path.exists("node_modules/handlebars"):
     print("Handlebars binary missing due to rebase past .gitignore fixup; fixing...")
     subprocess.check_call(["rm", "-rf", "node_modules/handlebars"])
-    subprocess.check_call(["yarn", "install", "--non-interactive", "--prefer-offline"])
+    YARN_BIN = "/srv/zulip-npm-cache/yarn/bin/yarn"
```

Comment:
Hmmm... Since `YARN_BIN` has already been symlinked to `/usr/bin/yarn`, why should a hardcoded path be specified whenever a script uses it?